packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1640,8 +1644,16 @@ describe('ReactDOMInput', () => {
       );
       const node = ReactDOM.findDOMNode(stub);
 
+      node.focus()
+
       setUntrackedValue.call(node, '2');
       dispatchEventOnNode(node, 'input');
+
+      // TODO: it is unclear why blur must be triggered twice,
+      // manual testing in the fixtures shows that the active element
+      // is no longer the input, however blur() + a blur event seem to
+      // be the only way to remove focus in JSDOM
+      node.blur()
```

Comment:
@gaearon this was the false positive case mentioned earlier. JSDOM doesn't update the active element on blur, only after blur() + a blur event. 

I have also walked through the number fixtures to verify that things continue to work as expected manually.